openadapt/config.py

```diff
@@ -132,35 +132,45 @@ def obfuscate(val, pct_reveal=0.1, char="*"):
     return rval
 
 
-
 _OBFUSCATE_KEY_PARTS = ("KEY", "PASSWORD", "TOKEN")
 if multiprocessing.current_process().name == "MainProcess":
     for key, val in dict(locals()).items():
         if not key.startswith("_") and key.isupper():
             parts = key.split("_")
             if (
-                any([part in parts for part in _OBFUSCATE_KEY_PARTS]) and
-                val != _DEFAULTS[key]
+                any([part in parts for part in _OBFUSCATE_KEY_PARTS])
+                and val != _DEFAULTS[key]
             ):
                 val = obfuscate(val)
             logger.info(f"{key}={val}")
 
 
-def filter_log_messages(data):
-    """
-    This function filters log messages by ignoring any message that contains a specific string.
+def filter_log_messages(data, max_count=1):
```

Comment:
Thanks @Mustaballer ! 

What do you think about `max_count=CONFIG.MAX_NUM_REPEAT_WARNINGS`?

Reply:
resolved in https://github.com/OpenAdaptAI/OpenAdapt/pull/389/commits/a43d467164ece4364b874d0e7b9b524bce342e45